### DISCUSSION TITLE
Cancel to list results found option

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [hsoft/dupeguru#1171] [Ideas]
I have a large image library and Dupeguru can take a large amount of time >24 hours to process a scan (on a 5950x  PCO @4.5GHz CPU usage 95%)

If I could stop a scan after an overnight 12-14 run without loosing the results it would enable me to process (ignore/delete/move) them and subsequent scans could be quicker

Alldup that use to find exact (byte-by-duplicates) lets me do this, as well as terminating after X matches found to add processing

**Comment by XtraCrispi** [comment]
Yes, please!  I'm currently in to day #3 of scanning two NAS units. I wish I could stop and save progress. It would also be nice if there were auto-save points (i.e. every 25% for scans over xx GB) so if the power goes out or my Windows machine sees fit to install updates and reboot, I don't lose everything.

Additionally, I wish there was a way to set reference directories post scan. For example, if between two NAS units or multiple locations I have a sorted folder and a 'junk' disorganized folder and I want to say keep all those in the sorted folder and flag the 'junk' items as the duplicates.
